auto-random.md

```diff
@@ -9,7 +9,7 @@ aliases: ['/docs-cn/dev/reference/sql/attributes/auto-random/']
 
 > **警告：**
 >
-> 当前 `AUTO_RANDOM` 属性为实验功能，**不建议在生产环境中使用**。在后续版本中，`AUTO_RANDOM` 的语法或语义可能会变化。
+> 当前 `AUTO_RANDOM` 属性于 v4.0.0 版本已为正式功能，**可以在生产环境中使用**。
```

Comment:
Should we simply remove these lines for v4.0 and master branch?

Reply:
> Should we simply remove these lines for v4.0 and master branch?

I think it's okay. At that time, I wanted to distinguish it from the experimental characteristics and give it a clear explanation.In future versions (such as 5.0) we can delete this line directly.